test-cases/artifacts/amazon2.yaml

```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
+ami_id_db_scylla: 'ami-099a8245f5daa82bf'
+append_scylla_setup_args: '--no-cpuscaling-setup'
+aws_root_disk_name_db: '/dev/xvda'
+aws_root_disk_size_db: 50
+cluster_backend: 'aws'
+instance_type_db: 'i3.large'
+instance_provision: "spot_low_price"
+logs_transport: 'ssh'
+n_db_nodes: 1
+n_loaders: 0
+n_monitor_nodes: 0
+nemesis_class_name: 'NoOpMonkey'
+region_name: 'eu-west-1'
+scylla_linux_distro: 'centos'
+scylla_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-3.3.repo'
+store_results_in_elasticsearch: False
+system_auth_rf: 1
+test_duration: 600
```

Comment:
The artifact-test won't take too much time, 600 mins is too big.
The artifact-tests of master took about 18 mins: https://jenkins.scylladb.com/job/scylla-master/job/artifacts/

30 mins should be enough for artifact-test test_duration.

Reply:
We had an issue with OEL job: #1889 . 30 minutes is not enough sometimes. But I agree that 10 hours is redundant for artifacts test. I'll change it to 60.